requirements.txt

```diff
@@ -1,5 +1,45 @@
-flake8==5.0.4
-flake8-annotations==2.9.1
-flake8-quotes==3.3.1
-flake8-variables-names==0.0.5
-pep8-naming==0.13.2
+attrs==24.2.0

```

Comment:
Why are there so many requirements, do you use them all?

Reply:
I did command pip freeze > requirements.txt and got this result